src/plugins/controls/public/control_group/component/control_group_sortable_item.tsx

```diff
@@ -106,7 +106,7 @@ const SortableControlInner = forwardRef<
         style={style}
       >
         <ControlFrame
-          enableActions={isEditable && draggingIndex === -1}
+          enableActions={draggingIndex === -1}
```

Comment:
The available actions are now determined based on their individual compatibility + the disabled actions from the control group - so, we don't need to know if the dashboard is in edit mode for this anymore.